client/src/App.jsx

```diff
@@ -10,6 +10,7 @@ import "@fontsource/roboto/700.css"
 import styled from 'styled-components';
 import GuestAdder from './GuestAdder.jsx';
 import {StarOutlined, StarTwoTone, StarFilled} from '@ant-design/icons';
+import 'regenerator-runtime/runtime';
```

Comment:
I think this might be doing the same thing as the package I'm using, @babel/plugin-transform-runtime? I'm able to place it once in my babel config file. Might feel better than having to import it into each of your files.